scripts/search_start_points.py

```diff
@@ -0,0 +1,59 @@
+from scipy.optimize import minimize
+import numpy as np
+import os
+import csv
+
+from scripts.models import Qlearning, RescorlaWagner
+from scripts.player import Estimator, RealPlayer
+
+
+def search_optimal_parameters(estimator):
+    optimal_alpha = []
+    optimal_T = []
+    optimal_func_value = []
+    alpha_list = np.arange(0.0, 1.0, 0.01)
+    T_list = np.arange(1, 10, 0.1)
+    for alpha in alpha_list:
+        for T in T_list:
+            optimal_params = minimize(estimator.log_likelihood_function, x0=np.array([T, alpha]), method='Nelder-Mead',
+                                      args=(-1.0,))
+            optimal_func_value.append(optimal_params.fun)
+            optimal_T.append(optimal_params.x[0])
+            optimal_alpha.append(optimal_params.x[1])
+    max_index, max_value = max(enumerate(optimal_func_value))
+    return optimal_T[max_index], optimal_alpha[max_index]
+
+
+def save_optimal_params(data_dir_path, model, new_filename):
```

Comment:
also huge duplication, we should thinkg about how to adjust data_utils functions for those purposes and simply use them with logic that is inside 'search_optimal_parameters'

Reply:
actually those are starting points and not optimal values!